platform/darwin/src/MGLLineStyleLayer.h

```diff
@@ -366,6 +366,50 @@ MGL_EXPORT
 */
 @property (nonatomic) MGLTransition lineGapWidthTransition;
 
+#if TARGET_OS_IPHONE
+/**
+ Defines a gradient with which to color a line feature. Can only be used with
+ GeoJSON sources that specify `"lineMetrics": true`.
```

Comment:
We’ll need to customize this documentation comment by adding en entry to platform/darwin/scripts/style-spec-overrides-v8.json:

> This property only has an effect on lines defined by an `MGLShapeSource` whose `MGLShapeSource.calculatesLineDistanceMetrics` property is set to `YES`.

Speaking of which, we’ll need to add that `calculatesLineDistanceMetrics` property to MGLShapeSource before developers will be able to use this property on iOS/macOS.

Reply:
#13000